rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
@kentcdodds I pushed a fix for that.
It feels a bit hacky. I guess we can probably fix that at build time directly via rollup. Not sure how tho.